File: src/main/java/net/kingdommc/darkages/numinoustreasury/item/action/NuminousOnEat.java
package net.kingdommc.darkages.numinoustreasury.item.action;

import org.bukkit.configuration.serialization.ConfigurationSerializable;
import org.bukkit.event.player.PlayerItemConsumeEvent;

public interface NuminousOnEat extends ConfigurationSerializable {
    void onEat(PlayerItemConsumeEvent event);
}

File: src/main/kotlin/net/arvandor/numinoustreasury/item/action/NuminousOnEat.kt
package net.arvandor.numinoustreasury.item.action

import org.bukkit.configuration.serialization.ConfigurationSerializable
import org.bukkit.event.player.PlayerItemConsumeEvent

interface NuminousOnEat : ConfigurationSerializable {
    fun onEat(event: PlayerItemConsumeEvent)
}
